hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/SubscriptionEndpoint.java

```diff
@@ -1,9 +1,6 @@
 package pl.allegro.tech.hermes.api.endpoints;
 
-import pl.allegro.tech.hermes.api.PatchData;
-import pl.allegro.tech.hermes.api.Subscription;
-import pl.allegro.tech.hermes.api.SubscriptionHealth;
-import pl.allegro.tech.hermes.api.SubscriptionMetrics;
+import pl.allegro.tech.hermes.api.*;
```

Comment:
\[Checkstyle\] ERROR: Using the '\.\*' form of import should be avoided \- pl\.allegro\.tech\.hermes\.api\.\*\.